pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -662,13 +662,21 @@ func (m *kubeGenericRuntimeManager) computePodResizeAction(pod *v1.Pod, containe
 	return true
 }
 
-func (m *kubeGenericRuntimeManager) doPodResizeAction(pod *v1.Pod, podStatus *kubecontainer.PodStatus, podContainerChanges podActions, result kubecontainer.PodSyncResult) {
+func (m *kubeGenericRuntimeManager) doPodResizeAction(pod *v1.Pod, podStatus *kubecontainer.PodStatus, podContainerChanges podActions) (result kubecontainer.PodSyncResult) {
+	resizePodResult := kubecontainer.NewSyncResult(kubecontainer.ResizePodSandbox, format.Pod(pod))
+	result.AddSyncResult(resizePodResult)
+	for resourceName, cInfos := range podContainerChanges.ContainersToUpdate {
+		for _, cInfo := range cInfos {
+			container := pod.Spec.Containers[cInfo.apiContainerIdx]
+			m.recordContainerEvent(pod, &container, "", v1.EventTypeNormal, events.StartingVerticalScalingPod, "Pod starts vertical scaling pod %s resource", resourceName)
```

Comment:
This should be called in updatePodContainerResources(...) function, which is where the actual attempt to resize will be done. It may not even get there for eg. if leading setPodCgroupConfig were to fail for some reason, and then we would have inaccurate events recorded.